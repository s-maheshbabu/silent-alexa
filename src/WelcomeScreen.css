/** Scaffolding **/
#welcome-screen {
  grid-column: 1 / 3;
  grid-template-rows: auto;
  height: calc(100vh - 60px);
  overflow: auto;
  text-align: center;
}
