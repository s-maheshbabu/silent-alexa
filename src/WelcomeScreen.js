import React from "react";
import "./WelcomeScreen.css";

export default function WelcomeScreen(props) {
  return (
    <div id="welcome-screen">
      <h1> WELCOME TO SILENT ALEXA </h1>
      <h1> Login to get started </h1>
    </div>
  );
}
