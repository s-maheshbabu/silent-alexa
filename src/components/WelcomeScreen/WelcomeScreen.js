import React from "react";
import "./WelcomeScreen.css";

export default function WelcomeScreen() {
  return (
    <div id="welcome-screen">
      <h1> WELCOME TO SILENT ALEXA (Under Construction) </h1>
      <h1> Login to get started </h1>
    </div>
  );
}
