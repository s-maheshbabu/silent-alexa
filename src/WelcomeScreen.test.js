import React from "react";
import { shallow } from "enzyme";
import WelcomeScreen from "WelcomeScreen";

it("renders without crashing", () => {
  const wrapper = shallow(<WelcomeScreen />);
  expect(wrapper).toMatchSnapshot();

  wrapper.unmount();
});
