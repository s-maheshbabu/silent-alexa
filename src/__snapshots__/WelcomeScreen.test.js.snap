// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  id="welcome-screen"
>
  <h1>
     WELCOME TO SILENT ALEXA (Under Construction) 
  </h1>
  <h1>
     Login to get started 
  </h1>
</div>
`;
